src/BUILD

```diff
@@ -1,11 +0,0 @@
-filegroup(
-    name = "core_srcs",
```

Comment:
You can't delete this, it's needed by sigh_webpack

Reply:
sigh_command uses //:all_srcs which includes core_srcs, which basically means bazel will create a symlink farm from repo_root into runfiles, which is similar to how run_in_repo root semantics works, only with more isolation.